src/index.html

```diff
@@ -2,17 +2,90 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
     <title>Stars</title>
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
```

Comment:
Keep your attributes correctly formatted. Since there are multiple attributes, each one should start on a new line with 2-space indentation related to the tag.